<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsEditor</class>
 <widget class="QWidget" name="ControlsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,4,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>controls tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="controlsTree">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="elemPropsWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>test ui</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="controlsView">
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>623</width>
           <height>357</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>test commands log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="commandsLogView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addGroupBtn">
        <property name="toolTip">
         <string>Add group</string>
        </property>
        <property name="text">
         <string>add group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addCommandBtn">
        <property name="toolTip">
         <string>Add control</string>
        </property>
        <property name="text">
         <string>add command</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addParamBtn">
        <property name="toolTip">
         <string>Add parameter</string>
        </property>
        <property name="text">
         <string>add command parameter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeBtn">
        <property name="toolTip">
         <string>Remove element</string>
        </property>
        <property name="text">
         <string>remove element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearLogBtn">
        <property name="text">
         <string>clear log</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>47</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
